sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample2_RecognizeReceiptsFromFile.cs

```diff
@@ -32,31 +32,75 @@ public async Task RecognizeReceiptsFromFile()
                 RecognizedReceiptCollection receipts = await client.StartRecognizeReceipts(stream).WaitForCompletionAsync();
                 foreach (var receipt in receipts)
                 {
-                    USReceipt usReceipt = receipt.AsUSReceipt();
+                    FormField merchantNameField;
+                    if (receipt.RecognizedForm.Fields.TryGetValue("MerchantName", out merchantNameField))
+                    {
+                        if (merchantNameField.Value.Type == FieldValueType.String)
+                        {
+                            string merchantName = merchantNameField.Value.AsString();
+
+                            Console.WriteLine($"    Merchant Name: '{merchantName}', with confidence {merchantNameField.Confidence}");
+                        }
+                    }
 
-                    string merchantName = usReceipt.MerchantName?.Value ?? default;
-                    DateTime transactionDate = usReceipt.TransactionDate?.Value ?? default;
-                    IReadOnlyList<USReceiptItem> items = usReceipt.Items ?? default;
-                    float subtotal = usReceipt.Subtotal?.Value ?? default;
-                    float tax = usReceipt.Tax?.Value ?? default;
-                    float tip = usReceipt.Tip?.Value ?? default;
-                    float total = usReceipt.Total?.Value ?? default;
+                    FormField transactionDateField;
+                    if (receipt.RecognizedForm.Fields.TryGetValue("TransactionDate", out transactionDateField))
+                    {
+                        if (transactionDateField.Value.Type == FieldValueType.Date)
+                        {
+                            DateTime transactionDate = transactionDateField.Value.AsDate();
 
-                    Console.WriteLine($"Recognized USReceipt fields:");
-                    Console.WriteLine($"    Merchant Name: '{merchantName}', with confidence {usReceipt.MerchantName.Confidence}");
-                    Console.WriteLine($"    Transaction Date: '{transactionDate}', with confidence {usReceipt.TransactionDate.Confidence}");
+                            Console.WriteLine($"    Transaction Date: '{transactionDate}', with confidence {transactionDateField.Confidence}");
+                        }
+                    }
 
-                    for (int i = 0; i < items.Count; i++)
+                    FormField itemsField;
+                    if (receipt.RecognizedForm.Fields.TryGetValue("Items", out itemsField))
```

Comment:
These `items` still correspond to the previously know model `USReceiptItem`  i.e `Quantity`, `Name` ..?